api/storage_service.hh

```diff
@@ -13,6 +13,7 @@
 #include <seastar/core/sharded.hh>
 #include "api.hh"
 #include "db/data_listeners.hh"
+#include "../table_info.hh"
```

Comment:
That better be "api/table_info.hh" so it wouldn't be relative to this source location

Reply:
And there's already a definition of it in api/storage_service.hh that can be included here....